modules/installing-ocp-agent-encrypt.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent-based-installer.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-encrypt_{context}"]
+= Optional: Encrypting the disk
+
+Use this procedure to encrypt your disk or partition while installing {product-title} with the Agent-based Installer.
+
+If you do not want to encrypt your disk or partition, proceed to the next section.
+
+.Prerequisites
+
+* You have created and configured the `install-config.yaml` and `agent-config.yaml` files, unless you are using ZTP manifests.
+
+* You have placed the `openshift-install` binary in a directory that is on your PATH.
+
+.Procedure
+
+. Use the following command to generate ZTP cluster manifests:
++
+[source,terminal]
+----
+$ openshift-install agent create cluster-manifests --dir <installation_directory>
+----
++
+[IMPORTANT]
+====
+If you have created the `install-config.yaml` and `agent-config.yaml` files, do not make further edits to those files after running this command.
+
+The `openshift-install agent create cluster-manifests` command generates ZTP cluster manifests that include the configurations made in those files.
+Once the ZTP files have been generated, the `install-config.yaml` and `agent-config.yaml` files will be ignored when the agent image is created.
```

Comment:
Is this technically accurate? I don't know whether these files are either deleted when `openshift-install agent create cluster-manifests` is run, or kept but ignored whenever the agent image is generated.